src/ui/components/Timeline/index.tsx

```diff
@@ -138,6 +143,9 @@ class Timeline extends Component<PropsFromRedux, { isDragging: boolean }> {
       focusRegion &&
       (mouseTime < focusRegion.startTime || mouseTime > focusRegion.endTime) &&
       !isFocusing;
+    const clickedOnNonLoadingRegion = nonLoadingTimeRanges.some(
+      r => r.start <= mouseTime && r.end >= mouseTime
```

Comment:
nit https://github.com/RecordReplay/devtools/blob/d7683b635fd8ea15717fa0c1e130e91c0b9e2d4c/src/ui/utils/timeline.ts#L149-L151

Reply:
Nice catch. Using it!

Reply:
Ah I would have, but you'll notice that the `nonLoadingRegions` is a different type from `TimeStampedPointRange[]`, since it doesn't have a point+time for the start and end. It only has the time. 

The reason is there's no easy way to get the last point in a recording. So it's hard to recreate the inverse of `loadedRegions.loading` with the points intact. But we know what the endTime is (thanks to zoomRegion), so that's what I end up using here.